ENGAGEHF/Resources/Localizable.xcstrings

```diff
@@ -89,6 +89,29 @@
         }
       }
     },
+    "DISCARD" : {
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Discard"
+          }
+        }
+      }
+    },
+    "Dismiss" : {
+
+    },
+    "DISMISS_BUTTON" : {
```

Comment:
As noted in the views, it would make sense to rather use human readable string keys in the long-run.